test/twofishes/client_test.rb

```diff
@@ -29,6 +29,90 @@ def self.geocode(location)
       assert result.parents
     end
 
+    it 'should accept max keyword argument' do
+      Twofishes::Client.expects(:thrift_client).returns(ZurichWithParents)
+      result = Twofishes::Client.geocode('Zurich', max: 5).first
+      assert result.parents
+    end
+
+    it 'should accept ll keyword argument' do
+      Twofishes::Client.expects(:thrift_client).returns(ZurichWithParents)
+      result = Twofishes::Client.geocode('Zurich', ll: "47,8").first
+      assert result.displayName
+
+      Twofishes::Client.expects(:thrift_client).returns(ZurichWithParents)
+      result = Twofishes::Client.geocode('Zurich', ll: [47,8]).first
+      assert result.displayName
+
+      Twofishes::Client.expects(:thrift_client).returns(ZurichWithParents)
+      result = Twofishes::Client.geocode('Zurich', ll: {:lng => 8, lat: 47}).first
+      assert result.displayName
+
+      Twofishes::Client.expects(:thrift_client).returns(ZurichWithParents)
+      result = Twofishes::Client.geocode('Zurich', ll: {"lng" => 8, lat: 47}).first
+      assert result.displayName
+    end
+
+    it 'should accept radius keyword argument' do
+      Twofishes::Client.expects(:thrift_client).returns(ZurichWithParents)
+      result = Twofishes::Client.geocode('Zurich', radius: 5000).first
+      assert result.parents
```

Comment:
This test should test that the radius attribute works, but it just tests if it returns results.